src/config/enUSText.js

```diff
@@ -316,11 +316,11 @@ var SettingsScreen = {
     Header: 'Delivery Limits',
     Info:
       'Maximum basal rate is the highest temporary basal rate Loop is allowed to set automatically.\n\nMaximum bolus is the highest bolus amount you can deliver at one time to cover carbs or bring down high glucose.',
-    MaxBasalRate: 'Maximum Basal Rate',
+    MaxBasalRate: 'Maximum Basal Rate, 5, U⁠/⁠hr',
```

Comment:
Short answer: It isn't targetable unless it is the full string.  The targeting uses the accessibility element which, I imagine, for actual accessibility reasons includes the string, value, and unit.  However, this is somewhat puzzling to me because that code has been around since Nov 2020, so why is this just failing now?  That's a mystery.

Reply:
Definitely strange, but thanks for explaining. Seems like detox is probably doing the "right" thing now.